src/app/store/api.js

```diff
@@ -0,0 +1,14 @@
+const URL = 'http://localhost:8000/api/v1/data';
+
+/* eslint-disable import/prefer-default-export */
+export async function getProbe(params) {
```

Comment:
This is called `getProbe`, but the `URL` is for the histogram data, not just the probe. Did you want the probe URL -- `/api/v1/probe/<name>`?

Reply:
oh good point. This should be `getProbeData` probably.